src/main/kotlin/exercise3/task4/TaxiParkTask.kt

```diff
@@ -10,29 +10,49 @@ package exercise3.task4
  * Find all the drivers who performed no trips.
  */
 internal fun TaxiPark.findFakeDrivers(): Set<Driver> {
-    TODO("Implement me!!!")
+    return this.allDrivers.minus(this.trips.map { it.driver }
+        .intersect(this.allDrivers))
```

Comment:
We do not need an `intersect` call here. 

Its enough to subtract `allDrivers - trips.map { it.drivers }` to get the result.